supplemental_ui/js/vendor/search.js

```diff
@@ -192,8 +195,15 @@ window.antoraLunr = (function (lunr) {
     var search = debounce(function () {
       searchIndex(index.index, index.store, searchInput.value)
     }, 100)
-    // TODO listen to blur, focus and input events
     searchInput.addEventListener('keydown', search)
+
+    // this is prevented in case of mousedown attached to SearchResultItem
+    searchInput.addEventListener('blur', function (e) {
+      e.target.value = ''
```

Comment:
I do agree that we should "close" (hide) the search result list when the user clicks on the page but please do not clear the search input. It should be an explicit action from the user.

```suggestion
```

Reply:
c'est commit